runtime/compiler/env/j9method.cpp

```diff
@@ -2366,6 +2380,20 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_OpaqueMethodBlock * aMethod, TR_Fron
       _jniTargetAddress = NULL;
       }
 
+   construct();
+   }
+
+#if defined(JITSERVER_SUPPORT)
+// protected constructor to be used by JITServer
```

Comment:
Could you please capitalize the beginning of sentences:
`Protected constructor to be used by JITServer`
`Had to reorder arguments to prevent ambiguity with above constructor`